summary/summary.py

```diff
@@ -109,7 +109,7 @@ def summary(plugin):
     reply['avail_out'] = avail_out.to_btc_str()
     reply['avail_in'] = avail_in.to_btc_str()
 
-    if plugin.fiat_per_btc:
+    if hasattr(plugin, 'fiat_per_btc'):
```

Comment:
We really should initialize this in `__init__` since this kind of static method call on an object is generally frowned upon.

Reply:
@cdecker like this 858de14 ?